pyproject.toml

```
@@ -0,0 +1,19 @@
+[tool.mypy]
+python_version = "3.12"
+pretty = true
+strict = true
+follow_imports = "silent"
+
+# Extra checks that aren't included in --strict
+enable_error_code = "ignore-without-code"
+extra_checks = true
+warn_unreachable = true
+exclude = [
```

oops, I guess a newline here would be helpful to clarify that this isn't an "extra check" (the comment only applies to the above block)

```suggestion

exclude = [
```